core/src/mos6502.h

```diff
@@ -63,9 +67,13 @@ class Mos6502 : public ICpu {
     // Returns an atom for the cpu pipeline for branching on a condition.
     std::function<void()> branch_on(std::function<bool()> condition);
 
-    // Returns a function which, when called, will write the given byte to a
-    // memory address. The address is fetched using absolute addressing mode.
-    std::function<void()> store_byte_abs_addr(uint8_t byte);
+    void create_store_instruction(Opcode opcode);
```

Comment:
This should probably also return a pipeline?

Reply:
Yes, I agree!